frontend/src/components/CopyURL.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { getCurrentURL } from "../utils";
+
+const CopyPostURL = ({ postId }) => {
+  const copyURL = () => {
+    let currentURL = getCurrentURL();
+
+    if (postId) {
+      currentURL = currentURL + `posts/${postId}`;
+    }
+
+    navigator.clipboard.writeText(currentURL);
+  };
+
+  return <button onClick={copyURL}>get Url</button>;
```

Comment:
suggestion => maybe change that text to smth like
```suggestion
  return <button onClick={copyURL}>Copy post url</button>;
```